smarts/__init__.py

```diff
@@ -19,7 +19,8 @@
 # THE SOFTWARE.
 
 
-import pkg_resources
+# import pkg_resources
 
 # The full version, including alpha/beta/rc tags
-VERSION = pkg_resources.get_distribution("smarts").version
+# VERSION = pkg_resources.get_distribution("smarts").version
+VERSION = "1.0.0"
```

Comment:
Is this meant to be left in?

Reply:
No, I was testing and I think this found its way into the changes. The main cause was that when you import `pkg_resources` it raises an error if you do not have the exactly compatible packages installed. This was causing issues for me when I was testing out packages.

Reply:
If possible I would get rid of it. There is a bit of a problem in that if we put the version in `smarts` then it fails to load but if we put it in here then `smarts` gets locked into requiring all packages to always be within the versions in the requirements.

Reply:
I have reverted this.